testing/test_aux.out

```diff
@@ -199,7 +202,13 @@ port-02 module_config modules
     "enabled": false
   },
   "dot1x": {
-    "enabled": true
+    "enabled": true,
```

Comment:
why is there here and in the other place?

Reply:
test_aux prints and compares module configs. I was considering dynamically editing the device_config.json file and adding GATEWAY_IP but this would get in the way.